packages/eslint-plugin-qwik/src/noPreventDefault.ts

```diff
@@ -0,0 +1,122 @@
+import { AST_NODE_TYPES, ESLintUtils } from '@typescript-eslint/utils';
+import type { ArrowFunctionExpression, BlockStatement, Expression, FunctionExpression, ObjectPattern } from '@typescript-eslint/types/dist/generated/ast-spec';
+
+const createRule = ESLintUtils.RuleCreator(() => 'https://github.com/BuilderIO/qwik/tree/main/packages/eslint-plugin-qwik');
```

Comment:
The url here points to the eslint plugin README, I could change this to the [Prevent Default Section](https://qwik.builder.io/docs/components/events/#prevent-default) of the docs instead, I think it could make more sense... although I would imagine the best would actually be to have more info in the plugin README(s) and eventually redirect from there to the docs.